.gitignore

```diff
@@ -15,3 +15,7 @@
 logfiles/
 *.mat
 *.tsv
+
+stimuli/Grahn2007_complex.txt
```

Comment:
perhaps we could keep these text files in the /stim folder? They are tiny, and don't need to be downloaded.

if you want to ignore .txt files, perhaps let's do:
*.txt
!stimuli/Grahn2007_simple.txt
!stimuli/Grahn2007_complex.txt

What do you think?

Reply:
they are in the stim folder. We decided to keep them there as default. Then participants download the .wav files.  
this issue happened because I accidentally deleted them in my folder and that's why I inserted again...